src/api/spec/models/staging/staging_project_spec.rb

```diff
@@ -11,9 +11,14 @@
   let(:staging_project) { staging_workflow.staging_projects.first }
 
   let!(:repository) { create(:repository, architectures: ['x86_64'], project: staging_project, name: 'staging_repository') }
+  let!(:repository_arch) { repository.repository_architectures.first }
+  let!(:architecture) { repository_arch.architecture }
   let!(:package) { create(:package_with_file, project: staging_project) }
 
-  let!(:status_report) { create(:status_report, checkable: repository) }
+  let!(:published_report) { create(:status_report, checkable: repository) }
+  let!(:build_report) { create(:status_report, checkable: repository_arch) }
```

Comment:
:thought_balloon: those are way better names :smile:

Reply:
That makes sense